Gems/AtomTressFX/Assets/Passes/HairParentPass.pass

```diff
@@ -25,17 +25,11 @@
                     "SlotType": "InputOutput",
                     "ScopeAttachmentUsage": "DepthStencil"
                 },
-                // Keep DepthLinear as input - used to set the size of the Head PPLL image buffer.
```

Comment:
Did you test this with PPLL approach?
It is crucial to test both render methods to verify that all allocations and timing remain intact.
In this case, the allocation of the PPLL buffer depends on this and I want to verify that removing this, you connected the allocation properly.

Reply:
Yes I tested both shortcut and PPLL.